CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Update meta-edison-bsp submodule with patch that fixes reboot issue on custom hardware [Sebastian]
```

Comment:
Please specify here the branch you have updated to and also the the commit revision